test/commands-tests/compatibility-tests.js

```diff
@@ -0,0 +1,136 @@
+
+        let temp = exec(cmd, executeOptions);
+        if (temp.stdout) {
+            temp.stdout.on('data', async (data) => {
+                let str = data.toString('utf8');
+                log += str;
+                if (str.trim() == 'Running tests...') {
+                    result = await exec(cliCommands.DOCKER_PS);
+                }
+            });
+        }
+
+        if (temp.stderr) {
+            temp.stderr.on('data', (data) => {
+                console.log('err:', data.toString('utf8'));
+            });
+        }
+
+        temp.on('exit', (code) => {
+            if (code !== 0) {
+                const msg = `Child process exited with code ${ code }`;
+                console.log(msg);
+                reject(msg)
+            }
+            
+            if (options && options.logs) {
+                resolve(log);
+            } else {
+                resolve(result);
+            }
+        });
+    });
+}
+
+describe('Compatibility tests', async function () {
+    let tempCWD = process.cwd();
+
+    before(async function () {
+        fs.ensureDirSync(`.${ testFolder }`);
+        await execute(constants.cliCommands.INIT, [], executeOptions);
+        process.chdir(executeOptions.cwd);
+    })
+
+    it('Docker images should be run with "latest" versions', async function () {
+        let result = await compatibilityCmd();
+
+        if (result && result.stdout) {
+            const isNodeLatestVersion = result.stdout.indexOf(`${ nodeConfig.dockerImage }:latest`) >= 0;
+            const isCompilerLatestVersion = result.stdout.indexOf(`${ compilerConfig.dockerImage }:latest`) >= 0;
+
+            assert.isOk(isNodeLatestVersion && isCompilerLatestVersion, 'Node is not running with latest version');
+            assert.isOk(isCompilerLatestVersion, 'Compiler is not running with latest version');
+        } else {
+            assert.isOk(false, 'Cannot get result of "docker ps" command');
+        }
+    })
+
+    it('Docker images should be run with "specific" versions', async function () {
+        const nodeVersion = 'v5.1.0';
+        const compilerVersion = 'v3.1.0';
+        
+        let result = await compatibilityCmd({ nodeVersion: nodeVersion, compilerVersion: compilerVersion });
+
+        if (result && result.stdout) {
+            const isNodeLatestVersion = result.stdout.indexOf(`${ nodeConfig.dockerImage }:${ nodeVersion }`) >= 0;
+            const isCompilerLatestVersion = result.stdout.indexOf(`${ compilerConfig.dockerImage }:${ compilerVersion }`) >= 0;
+
+            assert.isOk(isNodeLatestVersion && isCompilerLatestVersion, 'Node is not running with specific version');
+            assert.isOk(isCompilerLatestVersion, 'Compiler is not running with specific version');
+        } else {
+            assert.isOk(false, 'Cannot get result of "docker ps" command');
+        }
+    })
+
+    it('Tests should be run successfully and should not be running docker images', async function () {
```

Comment:
probably "and should stop node and compiler" is a better name, if that is the purpose of the test